backend/typescript/services/implementations/campService.ts

```diff
@@ -516,21 +516,6 @@ class CampService implements ICampService {
         throw new Error(`Camp' with campId ${campId} not found.`);
       }
 
-      // Update the campSessions by deleting all current sessions and creating new sessions
```

Comment:
After fixing that problem above regarding the validators, I noticed that campers were being deleted every time we viewed (updated) a camp. This is because of this part of the code which I deleted- shouldn't be a problem anywhere else since we don't use updateCampById for anything other than when we view a camp.

Reply:
I'm glad you caught this 😅 you're right- it is a good idea to seperate the logic of the two, so I noticed that we were calling `editCampById` both when on the camp overview (since the volunteer/coordinator editing component would call this), and when we saved or published a draft from the footer (which is what you caught). For the overview we can't have new sessions created, but for the drafting we do need new sessions, and honestly the best way I could find that works was just to create another (yet very similar) end point, both of which call `updateCampById` but now pass in a `createNewSessions` boolean arg which indicates whether we want new sessions or not. Feel free to leave your thoughts on this approach.